libs/design-system/src/panda/semanticColors.tokens.ts

```diff
@@ -30,16 +30,25 @@ export const LEGACY_COLOR_SEMANTIC_TOKENS = defineSemanticTokens.colors({
         value: { base: '{colors.legacy.B70}', _dark: '{colors.legacy.B40}' },
         type: 'color',
       },
-      /*
-       * disabled: {
-       *   value: { base: '{colors.legacy.mauve.30.light}', _dark: '{colors.legacy.mauve.30.dark}' },
-       *   type: 'color',
-       * },
-       * accent: {
-       *   value: { base: '{colors.legacy.blue.20.light}', _dark: '{colors.legacy.blue.20.dark}' },
-       *   type: 'color',
-       * },
-       */
+      // not actually legacy, but makes the merging of the two easier for now.
```

Comment:
These are from the new design system, but since our feedback colors are kind of all over the place and used much more situationally, I decided exposing these here and now was a bit easier.